test/plugins/test-trace-generic-pool.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+if (!process.env.GCLOUD_PROJECT) {
+  console.log('The GCLOUD_PROJECT environment variable must be set.');
+  process.exit(1);
+}
+
+var assert = require('assert');
+var common = require('./common');
+
+describe('generic-pool2', function() {
+  var ROOT_SPAN = 'root-span';
+  var CHILD_SPAN_1 = 'child-span-1';
+  var CHILD_SPAN_2 = 'child-span-2';
+
+  var api;
+  var genericPool;
+  before(function() {
+    api = require('../..').start({ samplingRate: 0, stackTraceLimit: 0 });
```

Comment:
This was unintentional.  I have removed `stackTraceLimit: 0` from both the generic-pool 2 and 3 tests.